code/modules/mining/ores_coins.dm

```diff
@@ -317,24 +317,25 @@ GLOBAL_LIST_INIT(sand_recipes, list(\
 /obj/item/coin
 	icon = 'icons/obj/economy.dmi'
 	name = "coin"
-	icon_state = "coin"
+	icon_state = "coin" //This is changed on initialization, the icon state is used to define which coin type to use.
 	flags_1 = CONDUCT_1
 	force = 1
 	throwforce = 2
 	w_class = WEIGHT_CLASS_TINY
 	custom_materials = list(/datum/material/iron = 400)
-	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR

```

Comment:
due to my coin system the only thing you need to do to add a new coin is put it in possible things to input in the coin generator
every new coin will now look the same as the default one and it doesnt make sense that a plastic or runite coin is bronze
also two headed coins are iron coins

Reply:
Fixed it, now coin types without a unique sprite will instead use the color system by default. This includes any newly-added coins.